packages/fxa-content-server/app/scripts/views/force_auth.js

```diff
@@ -139,6 +139,11 @@ var View = SignInView.extend({
 
     return this.navigate('signup', {
       error: AuthErrors.toError('DELETED_ACCOUNT'),
+      // account is for the email-first signup flow, without an account
+      // the signup page redirects to index
+      account,
+      // forceEmail is a hint to the signup page that the user should not
```

Comment:
I am probably missing something but in what cases would we want to force emails for a user signup?

Reply:
If a user comes from Firefox's settings page, /settings is opened with a uid/email. If the account for that email no longer exists, we send the user to force_auth to sign up again.